contributing/workflow/bug_triage_guidelines.rst

```diff
@@ -58,16 +59,19 @@ The following labels are currently defined in the Godot repository:
 -  *Discussion*: the issue is not consensual and needs further
    discussion to define what exactly should be done to address the
    topic.
--  *Documentation*: issue related to the documentation. Mainly to request
-   enhancements in the API documentation. Issues related to the ReadTheDocs
-   documentation should be filed on the
+-  *Documentation*: related to the documentation. PRs with this label improve the
+   class reference. Issues with this label are not bugs, but limitations to be
+   further documented. Often paired with *Discussion*. Issues related to the
```

Comment:
We actually consider wrong documentation bugs too, and those issues do get labeled both with "bug" and "documentation", so "issues with this label are not bugs" isn't entirely technically correct.